README.md

```diff
@@ -124,6 +128,20 @@ secretDescriptor:
       name: password
 ```
 
+The following IAM policy allows a user to access parameters matching `prod-*`.
```

Comment:
Could we also give an example IAM policy for secretsManager in the README?

Reply:
good point, i added it.